pkg/kube/inject/webhook.go

```diff
@@ -1110,13 +1111,15 @@ func (wh *Webhook) inject(ar *kube.AdmissionReview, path string) *kube.Admission
 	if pod.ObjectMeta.Namespace == "" {
 		pod.ObjectMeta.Namespace = req.Namespace
 	}
-	log.Infof("Sidecar injection request for %v/%v", req.Namespace, podName)
+
+	log = log.WithLabels("ns", req.Namespace, "pod", pod.Namespace+"/"+podName)
```

Comment:
why do we need `ns`? its already part of the `pod` label?

Reply:
Could they be different? See L1112

Reply:
Thanks, removed `ns`.